pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/LLCSegmentCompletionHandlers.java

```diff
@@ -135,7 +135,7 @@ public String segmentConsumed(@QueryParam(SegmentCompletionProtocol.PARAM_INSTAN
 
     SegmentCompletionProtocol.Response response = _segmentCompletionManager.segmentConsumed(requestParams);
     final String responseStr = response.toJsonString();
-    LOGGER.info("Response to segmentConsumed:{}", responseStr);
+    LOGGER.info("Response to segmentConsumed for segment {} is :{}", segmentName, responseStr);
```

Comment:
Suggest changing the message to: "Response to segmentConsumed for segment: '{}' is: {}"
(Add single quote around segment name so we know if there are leading or trailing spaces, also for the ':', no space before and one space after)